_api-reference/security/configuration/get-configuration.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: default
+title: Get Configuration API
+parent: Configuration APIs
+grand_parent: Security APIs
+nav_order: 40
+---
+
+# Get Security Configuration API
+**Introduced 1.0**
+{: .label .label-purple }
+
+The Get Security Configuration API retrieves the current Security plugin configuration. This configuration includes authentication domains, authorization settings, and other security-related configurations that determine how users and requests are authenticated and authorized in your OpenSearch cluster.
```

Comment:
The Security Configuration API is more focused on setting on authentication and general settings rather than authorization. I would say that Roles and Roles Mappings are the real authorization APIs where users are granted permission to do stuff.

As part of the `authc:` section of the configuration an administrator often needs to specify a `subject_key` and `roles_key` to tell OpenSearch how to extract `backend_roles` which can be used in roles mappings to map to OpenSearch roles.

There is an `authz:` section that can be added to the security configuration, but that is specific to extracting backend roles from LDAP systems.

I'm also wondering if we should drop the word `plugin` from `...the current Security plugin configuration`

Reply:
Noted on the Plugin language. I'm tweaking that now.

I'll add the additional information about authc and authz on the Create Configuration page or the Configuration home page.